lib/telemetry.js

```diff
@@ -34,7 +35,7 @@ const unactionableErrors =
 
 setInterval(() => {
   writeClient.flush().catch(err => {
-    if (!unactionableErrors.test(String(err))) {
+    if (!unactionableErrors.test(inspect(err))) {
```

Comment:
How about applying the check on the error code string directly?

```js
if (unactionableErrors.test(String(err))) return;
if (err?.code && unactionableErrors.test(String(err.code))) return;
Sentry.captureException(err)
```

Reply:
I really like this, even simpler!